Dropped Django 3.0 and 3.1 support.

hramezani opened the pull request:
* Dropped support for Django 3.0 and 3.1.

* Confirmed support for Python 3.10.

felixxm commented:
@hramezani Thanks :+1: Can you update the [`CHANGELOG`](https://github.com/django-request/django-request/blob/4b9c7b22f26338d2c93110477aa44041b1c5ddb4/CHANGELOG.md?plain=1#L7) and [`README`](
https://github.com/django-request/django-request/blob/4b9c7b22f26338d2c93110477aa44041b1c5ddb4/README.md?plain=1#L33-L34)?

hramezani commented:
@felixxm it's ready. take a look if you have time.

hramezani commented:
@felixxm I updated the patch. now it contains:
* Dropped support for Django < 3.2.

* Confirmed support for Python 3.10.

* Confirmed support for Django 4.0.

felixxm commented:
@hramezani I don't want to drop support for Django 2.2, at least for now.

felixxm commented:
I fixed `USE_TZ` deprecation warning in #232.

hramezani commented:
> @hramezani I don't want to drop support for Django 2.2, at least for now.

It's suggested by Tim. 
https://github.com/jazzband/django-hosts/pull/141#discussion_r766116557

felixxm commented:
> > @hramezani I don't want to drop support for Django 2.2, at least for now.
> 
> It's suggested by Tim. [jazzband/django-hosts#141 (comment)](https://github.com/jazzband/django-hosts/pull/141#discussion_r766116557)

I don't want to drop support for Django 2.2 before April 2022, it's still in extended support.

hramezani commented:
I will change the patch to drop Django 3.0 support

felixxm commented:
> I will change the patch to drop Django 3.0 support

... and 3.1.

felixxm commented:
@hramezani Thanks :+1:
